### ISSUE TITLE
Dragging outside map container

### ISSUE THREAD

**Issue by lennerd** [maptalks/maptalks.js#1517]
**maptalks's version and what browser you use?**
----
0.49.5

**Issue description**
----
When dragging a maptalks map outside its DOM container the drag interaction is stopped. Might be subjective but feels somehow weird and interruptive. Other map libraries like OpenLayers or Leaflet are supporting dragging like this.

Are there any options or future plans to support this?

**Please provide a reproduction URL (on any jsfiddle like site)**
----
Basically any maptalks map. For example: https://maptalks.org/examples/en/map/load/

**Comment by fuzhenn** [comment]
It's an expected behavior, because with some UI libraries, out-of-container dragging sometimes goes out of control.
Thanks for your advice, I will look into this if more ppl propose the same.
Closing this for now.
